src/views/product-landing/server.ts

```diff
@@ -1,14 +1,19 @@
+import fs from 'fs'
+import path from 'path'
 import slugify from 'slugify'
-// TODO: need to discuss from whence we should
-// TODO: source content down the road. For now,
-// TODO: sourcing from JSON for demo purposes.
-import TEMP_CONTENT from './content.json'
 
-const productName = 'Waypoint'
-const productSlug = 'waypoint'
-
-async function getStaticProps(): Promise<$TSFixMe> {
-  const CONTENT = TEMP_CONTENT as $TSFixMe
+async function getStaticProps({
+  product,
+  contentJsonFile,
+}: {
+  product: { slug: string; name: string }
+  contentJsonFile: string
+}): Promise<$TSFixMe> {
+  // TODO: need to discuss from whence we should
+  // TODO: source content down the road. For now,
+  // TODO: sourcing from JSON for demo purposes.
```

Comment:
Want to link the Asana task here for easy reference? https://app.asana.com/0/1100423001970639/1201631159784193/f